CHANGELOG.md

```diff
@@ -3,3 +3,9 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.0.0] - 2018-08-28
```

Comment:
Did you tag it already?

Reply:
No I assumed I would tag it once it hit master.